### PR TITLE
Changed line height

### PR DESCRIPTION
**Pull request by komarovska**
I changed something

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -53,7 +53,7 @@ section {
 
 h1 {
   font-size: 96px;
-  line-height: 100px;
+  line-height: 101px;
```
Comment (komarovska) [inline style.css:56]: What does this change mean? Why 101px?

File: index.html [hunk 2]
```diff
@@ -48,8 +48,8 @@ <h3 class="about_heading"><i>Musical artist</i>, live performer, record producer
         <p class="text-ident">When you listen to Sebastian Mullaert`s music you will feel something enchanting, a
           resonance that delivers
           truth,
-          integrity and reminds
-          us to simply be.</p>
+          integrity and <span>reminds</span>
+          us to simply be...</p>
```
Comment (komarovska) [inline index.html:52]: Are you sure this change is needed?

File: style.css (hunk 1 above)
Comment (ivanzhu7) [inline style.css:56]: I think it would be better 110px.

File: style.css [hunk 3]
```diff
@@ -53,7 +53,7 @@ section {
...
   font-weight: 100;
   padding: 75px 0;
   text-align: center;
```
Comment (Pshesh) [inline style.css:59]: Just a comment

File: index.html (hunk 2 above)
Comment (Nadiia85) [reply index.html:52]: its great idea

File: style.css [hunk 4]
```diff
@@ -53,7 +53,7 @@ section {
 
 h1 {
   font-size: 96px;
```
Comment (Lidko) [inline style.css:55]: Why 96px?

Comment (komarovska) [reply style.css:55]: I thought it would look good

File: style.css (hunk 1 above)
Comment (komarovska) [reply style.css:56]: @Pshesh what do you think?

**Review by Pshesh — APPROVED** [review]

**Review by Lidko — APPROVED** [review]

**Review by ivanzhu7 — APPROVED** [review]

**Pull request merged**